cmd/trade.go

```diff
@@ -192,30 +193,32 @@ func init() {
 		)
 		// --- end initialization of objects ---
 
-		log.Printf("validating trustlines...\n")
-		validateTrustlines(client, &botConfig)
-		log.Printf("trustlines valid\n")
+		l.Info("validating trustlines...")
+		validateTrustlines(l, client, &botConfig)
+		l.Info("trustlines valid")
 
 		// --- start initialization of services ---
 		if botConfig.MonitoringPort != 0 {
 			go func() {
-				e := startMonitoringServer(botConfig)
+				e := startMonitoringServer(l, botConfig)
 				if e != nil {
-					log.Println()
-					log.Printf("unable to start the monitoring server or problem encountered while running server: %s\n", e)
+					l.Info("")
+					l.Info("unable to start the monitoring server or problem encountered while running server:")
+					logger.Fatal(l, e)
```

Comment:
here too -- logger.Fatal will cause us to exit and the next line where we delete all offers won't run